packages/fxa-settings/src/pages/InlineTotpSetup/index.tsx

```diff
@@ -0,0 +1,397 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import classNames from 'classnames';
+import { MozServices } from '../../lib/types';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { useFtlMsgResolver } from '../../models';
+import { logViewEvent, logPageViewEvent } from '../../lib/metrics';
+import { ReactComponent as TwoFactorImg } from '../Signin/SigninTotpCode/graphic_two_factor_auth.svg';
+import InputText from '../../components/InputText';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+
+type InlineTotpSetupProps = {
+  code: string;
+  serviceName?: MozServices;
+};
+
+export const InlineTotpSetup = ({
+  code,
+  serviceName,
+}: InlineTotpSetupProps) => {
+  logPageViewEvent('inline-totp-setup', { entrypoint_variation: 'react' });
+
+  /*
+   * TODO:
+   *  - Write functionality to get TOTP token from account
+   *  - get secret and QR code image src from totp token
+   *  - add in success and error messages (with Banner component)
+   *  - add in actions for `Cancel` `Ready` etc
+   *  - fetch code here if that's preferable
+   */
+
+  const ftlMsgResolver = useFtlMsgResolver();
+  const localizedSecurityCodePlaceholder = ftlMsgResolver.getMsg(
+    'inline-totp-setup-security-code-placeholder',
+    'Security code'
+  );
+  const localizedQRCodeAltText = ftlMsgResolver.getMsg(
+    'inline-totp-setup-qr-code-alt-text',
```

Comment:
I couldn't find `inline-totp-setup-qr-code-alt-text` in an .ftl either:

```sh
git grep -n "inline-totp-setup-qr-code-alt-text" packages | cat

packages/fxa-settings/src/pages/InlineTotpSetup/index.tsx:38:    'inline-totp-setup-qr-code-alt-text',
```

but interesting (or not, you choose), I did find a similar string with the ID `tfa-qa-code-alt` in packages/fxa-settings/src/components/Settings/PageTwoStepAuthentication/en.ftl.

```sh
git grep -n "Use the code" -- *.ftl | cat

packages/fxa-settings/src/components/Settings/PageTwoStepAuthentication/en.ftl:26:tfa-qa-code-alt = Use the code { $secret } to set up two-step authentication in supported applications.
```

Reply:
Fixed!